collaborating/building.md

```diff
@@ -0,0 +1,20 @@
+### Building
+
+> [!NOTE]\
+> Ensure all the packages have been installed before building the project.
+To build different components of the IFRC Go project, use the following commands.
+
+**IFRC Go Web Application**
+  ```bash
+  yarn build
+  ```
+
+**IFRC Go UI Library**
```

Comment:
Let's use `GO` instead of `Go` everywhere. 
```suggestion
To build different components of the IFRC GO project, use the following commands.

**IFRC GO Web Application**
  \```bash
  yarn build
  \```

**IFRC GO UI Library**
```
> NOTE: Don't use \ , used to show suggestion markdown. thanks